src/script/lua_api/l_mainmenu.cpp

```diff
@@ -933,6 +933,17 @@ int ModApiMainMenu::l_get_video_drivers(lua_State *L)
 	return 1;
 }
 
+/******************************************************************************/
+int ModApiMainMenu::l_get_language(lua_State *L)
+{
+	std::string lang = gettext("LANG_CODE");
+	if (lang == "LANG_CODE")
+		lang = "en";
```

Comment:
usually this line is `lang = "";`

would also save you from the explicitly handling this in the builtin code